src/main.js

```diff
@@ -23,6 +24,9 @@ const app = new Vue({
   data: {
     user: null
   },
+  computed: {
+    ...mapGetters(['showSearchResults'])
+  },
```

Comment:
This isn't related to this change at all, it's something I forgot to add in another PR and the page breaks without it.